test/dlc_tests/container_tests/bin/benchmark/run_pytorch_inductor_training_benchmark_gpu_huggingface

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[0])' | tr -d "'")
+if [ "$PYTHON_VERSION" -eq 2 ]
+then
+  exit 0
+fi
+HOME_DIR=/test/benchmark
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs
+
+mkdir -p ${HOME_DIR}
+mkdir -p ${BIN_DIR}
+mkdir -p ${LOG_DIR}
+
+python -c "import sys;
+assert sys.version_info < (3, 7);"
+PYTHON_VERSION_CHECK=`echo $?`
+
+set -e
+rm -rf $BIN_DIR/pytorch
+# tmp fix for numpy deprecations, PR raised: https://github.com/HewlettPackard/dlcookbook-dlbs/pull/19
+git clone https://github.com/pytorch/pytorch.git $BIN_DIR/pytorch
```

Comment:
use `--single-branch --depth 1` to speed things up. The PT repo is > 1GB

Reply:
Fixed it.